ol3-viewer/src/ome/ol3/controls/ScaleBar.js

```diff
@@ -14,21 +14,31 @@ ome.ol3.controls.ScaleBar = function(opt_options) {
     if (typeof(opt_options) !== 'object') opt_options = {};
 
     /**
-	 * list available units for display incl. symbol and
-     * micron multiplicator
-	 *
-	 * @type {Array.<objects>}
-	 * @private
-	 */
+     * list available units for display incl. symbol, threshold for usage
+     * and micron multiplication factor
+     *
+     * @type {Array.<objects>}
+     * @private
+     */
     this.UNITS = [
-        { unit: 'micron', multiplier: 1, symbol: 'µm'},
-        { unit: 'millimeter', multiplier: 1000, symbol: 'mm'}];
+        { unit: 'angstrom',
```

Comment:
Unfortunately this is part of the ol3-viewer and there is no shared code as such, only eventbus communication. Also I'm not too sure how it will be reused since the default unit is micron (as it seems from the code). At most I'd get the symbol out of a shared definition, right?